network-api/networkapi/buyersguide/pagemodels/products/base.py

```diff
@@ -206,12 +269,128 @@ def specific(self):
         use_filename=True
     )
 
+    worst_case = models.CharField(
+        max_length=5000,
+        help_text="What's the worst thing that could happen by using this product?",
+        blank=True,
+    )
+
+    # What is required to sign up?
+
+    signup_requires_email = ExtendedYesNoField(
+        help_text='Does this product requires providing an email address in order to sign up?'
+    )
+
+    signup_requires_phone = ExtendedYesNoField(
+        help_text='Does this product requires providing a phone number in order to sign up?'
+    )
+
+    signup_requires_third_party_account = ExtendedYesNoField(
+        help_text='Does this product require a third party account in order to sign up?'
+    )
+
+    signup_requirement_explanation = models.TextField(
+        max_length=5000,
+        blank=True,
+        help_text='Describe the particulars around sign-up requirements here.'
+    )
+
+    # What data does it collect?
+
+    personal_data_collected = models.TextField(
+        max_length=5000,
+        blank=True,
+        help_text='What kind of personal data does this product collect?'
+    )
+
+    biometric_data_collected = models.TextField(
+        max_length=5000,
+        blank=True,
+        help_text='What kind of biometric data does this product collect?'
+    )
+
+    social_data_collected = models.TextField(
+        max_length=5000,
+        blank=True,
+        help_text='What kind of social data does this product collect?'
+    )
+
+    # How does it use this data?
+
+    how_does_it_use_data_collected = models.TextField(
+        max_length=5000,
+        blank=True,
+        help_text='How does this product use the data collected?'
+    )
+
+    data_collection_policy_is_bad = models.BooleanField(
+        default=False,
+    )
+
+    how_can_you_control_your_data = models.TextField(
+        max_length=5000,
+        blank=True,
+        help_text='How does this product let you control your data?'
+    )
+
+    data_control_policy_is_bad = models.BooleanField(
+        default=False,
+    )
+
+    # Company track record
+
+    track_record_choices = [
+        ('Great', 'Great'),
+        ('Average', 'Average'),
+        ('Needs Improvement', 'Needs Improvement'),
+        ('Bad', 'Bad')
```

Comment:
How is this going to be localized? Do we need a template tag similar to https://github.com/mozilla/foundation.mozilla.org/blob/master/network-api/networkapi/buyersguide/templatetags/yes_no.py#L19 ?

Reply:
correct.

Reply:
filed https://github.com/mozilla/foundation.mozilla.org/issues/5292 so we don't forget to do that!